scripts/generate-openapi-entities.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+if [ $# -ne 1 ]; then
+    echo "USAGE: <output path>"
+    echo "EXAMPLE: pkg/sdk/entity"
+
+    exit 1
+fi
+
+path=$1
+
+# download OpenAPI specs
+curl -s -O https://api.corbado.com/docs/api/openapi/common.yml
```

Comment:
we first need to expose `common.yml` because it's unavailable right now.

Reply:
we use bundle (https://redocly.com/docs/cli/commands/bundle/) in our build process so the common.yml part is already included in the final yml. Thus it should already work right now